### PR TITLE
Add indexes

### PR DESCRIPTION
**Pull request by skjnldsv**
Increase performances

### CODE REVIEW

File: lib/Migration/Version0010Date20190000000007.php [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 namespace OCA\Forms\Migration;
 
-use Doctrine\DBAL\Types\Type;
+use Doctrine\DBAL\Types;
```
Comment (nickvergessen) [inline lib/Migration/Version0010Date20190000000007.php:26]:
```suggestion
use Doctrine\DBAL\Types\Types;
```

File: lib/Migration/Version020200Date20210120082018.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+namespace OCA\Forms\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+/**
+ * Auto-generated migration step: Please modify to your needs!
+ */
```
Comment (nickvergessen) [inline lib/Migration/Version020200Date20210120082018.php:14]:
```suggestion
```

File: lib/Migration/Version020200Date20210120082018.php [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
```
Comment (nickvergessen) [inline lib/Migration/Version020200Date20210120082018.php:4]: header missing?

**Review by nickvergessen — APPROVED** [review]

File: lib/Migration/Version020200Date20210120082018.php [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+		if ($schema->hasTable('forms_v2_answers')) {
+			$schema->getTable('forms_v2_answers')
+			->addIndex(['submission_id'], 'forms_answers_submission')
+			->addIndex(['question_id'], 'forms_answers_question');
+		}
+
+		if ($schema->hasTable('forms_v2_forms')) {
+			$schema->getTable('forms_v2_forms')
+			->addIndex(['hash'], 'forms_forms_hash')
```
Comment (jotoeri) [inline lib/Migration/Version020200Date20210120082018.php:56]: @skjnldsv Just recognized, there is already an index on hash - Probably remove this line again?
grafik

File: lib/Migration/Version0010Date20190000000007.php (hunk 1 above)
Comment (jotoeri) [reply lib/Migration/Version0010Date20190000000007.php]: Are these available on DBAL<3? So are they for NC<21?

File: lib/Migration/Version020200Date20210120082018.php (hunk 4 above)
Comment (skjnldsv) [reply lib/Migration/Version020200Date20210120082018.php:56]: Was there? Ah, damn.
Can you open a pr for it?

**Pull request merged**